docs/index.html

```diff
@@ -64,8 +64,12 @@ <h1 class="sidebar-header">Lifespan filter<span class="sidebar-close"><i class="
               <input type="number" id="to">
             </div>
             <div class="lifespan-controls">
-              <button id="lifespan-pause"><i class="fas fa-pause"></i></button>
-              <button id="lifespan-play"><i class="fas fa-play"></i></button>
+              <p>To play a timelapse, select the starting year using the "From" handle, then specify window size by adjusting the "To" handle. 
+                The specified window will move forward in with the velocity specified below.</p>
```

Comment:
Edit the text as 
"To play a timelapse, select the starting year using the "From" handle, then specify window size by adjusting the "To" handle.  The specified window will move forward in **time** with the velocity specified below **until the year 2021**.